File: 2798-number-of-employees-who-met-the-target/Solution.py
# https://leetcode.com/problems/number-of-employees-who-met-the-target/submissions/1015864593/
class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours: List[int], target: int) -> int:
        return len(list(filter(lambda x: x >= target, hours)))

File: 2798-number-of-employees-who-met-the-target/Solution.java
// https://leetcode.com/problems/number-of-employees-who-met-the-target/submissions/1015860295/
class Solution {
    public int numberOfEmployeesWhoMetTarget(int[] hours, int target) {
        return (int) Arrays.stream(hours).filter(item -> item >= target).count();
    }
}
